### PR TITLE
Expand docstrings

### PR DESCRIPTION
**Pull request by BehindLoader**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -12,7 +12,11 @@
 
 
 async def create_connection() -> None:
-    """Create rabbitmq connection."""
+    """Create RabbitMQ connection.
+
+    Make async RabbitMQ connection and
+    bind callback for new messages in queue
```
Comment (IrvingWash) [inline app.py]:
```suggestion
    bind callback for new messages in the queue
```

Comment (BehindLoader) [inline app.py]: Add dot in the end of line.

File: src/builder.py [hunk 2]
```diff
@@ -11,15 +11,24 @@ class MinecraftBuilder(object):
...
         self.storage = BytesIO()
 
     async def prepare_server(self):
-        """Init pure server instance and save it to `self.storage`."""
+        """Init Minecraft server.
+
+        Download pure Minecraft server without any mods and settings from FTP
+        and save it to `self.storage` class attribute for work with him to:
+        - save mods to him
+        - update settings
```
Comment (BehindLoader) [inline src/builder.py]: Add dot in the end of line.

File: src/assets.py [hunk 3]
```diff
@@ -1,4 +1,7 @@
-"""Assembly assets."""
+"""Project assets.
+
+These assets helps for delegating task for necessary minecraft server builder.
```
Comment (IrvingWash) [inline src/assets.py]:
```suggestion
These assets help to delegate a task for a specific minecraft server builder.
```

File: src/builder.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-"""Module for building server objects."""
+"""Module for build game servers and save it to FTP."""
```
Comment (IrvingWash) [inline src/builder.py]:
```suggestion
"""Module for building game servers and saving them to FTP."""
```

File: src/builder.py [hunk 5]
```diff
@@ -11,15 +11,24 @@ class MinecraftBuilder(object):
     """Minecraft: Java Edition server building."""
 
     def __init__(self, options):
-        """Initializate builder with specified options."""
+        """Initializate Minecraft server builder.
```
Comment (IrvingWash) [inline src/builder.py]:
```suggestion
        """Initialize a Minecraft server builder.
```

File: src/builder.py [hunk 6]
```diff
@@ -11,15 +11,24 @@ class MinecraftBuilder(object):
...
         self.version = options['version']
 
         self.storage = BytesIO()
 
     async def prepare_server(self):
-        """Init pure server instance and save it to `self.storage`."""
+        """Init Minecraft server.
+
+        Download pure Minecraft server without any mods and settings from FTP
+        and save it to `self.storage` class attribute for work with him to:
```
Comment (IrvingWash) [inline src/builder.py]:
```suggestion
        and save it to `self.storage` class attribute to work with it to:
```

File: src/builder.py [hunk 7]
```diff
@@ -11,15 +11,24 @@ class MinecraftBuilder(object):
...
+        - save mods to him
```
Comment (IrvingWash) [inline src/builder.py]:
```suggestion
        - save mods into it
```

File: src/builder.py [hunk 8]
```diff
@@ -36,7 +45,11 @@ def __init__(self, options):
                 self.storage.seek(0)
 
     async def stor_server(self):
-        """Save ready server instance to FTP server."""
+        """Upload local Minecraft server instance to FTP.
+
+        Upload ready Minecraft server instance from `self.storage` attribute
```
Comment (IrvingWash) [inline src/builder.py]:
```suggestion
        Upload prepared Minecraft server instance from `self.storage` attribute
```

File: src/builder.py [hunk 9]
```diff
@@ -49,6 +62,15 @@ def __init__(self, options):
                 await stream.write(self.storage.read())
 
     async def build(self):
-        """Build minecraft server."""
+        """Minecraft server build pipeline.
+
+        This method will be called from lowest-level module `Task`
+        in method `Task.recieve(cls, message)`.
+        This method makes the Minecraft server by calling helper methods.
```
Comment (IrvingWash) [inline src/builder.py]:
```suggestion
        This method creates a Minecraft server by calling helper methods.
```

File: src/builder.py [hunk 10]
```diff
@@ -49,6 +62,15 @@ def __init__(self, options):
...
+
+        Algorithm in method deals with the following tasks:
+        - download server from FTP
+        - upload ready Minecraft server to FTP.
```
Comment (IrvingWash) [inline src/builder.py]:
```suggestion
        - upload prepared Minecraft server to FTP.
```

File: src/connection.py [hunk 11]
```diff
@@ -13,7 +13,11 @@
     prefetch_count: int,
     queue_name: str,
 ) -> Connection:
-    """Create connection and bind processing messages to callback."""
+    """Create connection with RabbitMQ service.
+
+    Create connection with RabbitMQ service and bind callback method
```
Comment (IrvingWash) [inline src/connection.py]:
```suggestion
    Create a connection with RabbitMQ service and bind callback method
```

File: src/settings.py [hunk 12]
```diff
@@ -1,4 +1,8 @@
-"""Project settings."""
+"""Project settings.
+
+Project settings variables are being taken from environment variables that
```
Comment (IrvingWash) [inline src/settings.py]:
```suggestion
Project settings variables are taken from environment variables which are
```

File: src/task.py [hunk 13]
```diff
@@ -1,20 +1,23 @@
-"""Module for class Task."""
+"""Module for gelegating tasks that are recieved from RabbitMQ queue."""
```
Comment (IrvingWash) [inline src/task.py]:
```suggestion
"""Module for delegating tasks that are received from RabbitMQ queue."""
```

File: src/task.py [hunk 14]
```diff
@@ -1,20 +1,23 @@
...
 
 from json import loads
 
 from src.assets import workers
 
 
 class Task(object):
-    """Task delegating."""
+    """Delegate and run task which are taken from RabbitMQ queue."""
```
Comment (IrvingWash) [inline src/task.py]:
```suggestion
    """Delegate and run a task which is taken from RabbitMQ queue."""
```

File: src/task.py [hunk 15]
```diff
@@ -1,20 +1,23 @@
...
-        """Task init method."""
+        """Parse RabbitMQ message and save it to class attribute."""
         self.text = message.body.decode('UTF-8')
         self.json = loads(self.text)
 
     def delegate(self):
-        """Get the necessary builder and create worker."""
+        """Get necessary server builder.
+
+        Get game server builder from `assets` by name and make instance of it.
```
Comment (IrvingWash) [inline src/task.py]:
```suggestion
        Get the game server builder from `assets` by name and make an instance of it.
```

File: src/task.py [hunk 16]
```diff
@@ -23,7 +26,12 @@ def delegate(self):
 
     @classmethod
     async def recieve(cls, message):
-        """Recieve message from mq."""
+        """Process RabbitMQ message.
+
+        Recieve message from RabbitMQ, parse it, get necessary builder for
```
Comment (IrvingWash) [inline src/task.py]:
```suggestion
        Receive a message from RabbitMQ, parse it, get the specific builder for
```

File: src/task.py [hunk 17]
```diff
@@ -23,7 +26,12 @@ def delegate(self):
...
+        this task, set up it and run.
```
Comment (IrvingWash) [inline src/task.py]:
```suggestion
        this task, set it up and run.
```

File: src/task.py [hunk 18]
```diff
@@ -23,7 +26,12 @@ def delegate(self):
...
+        This method are binded like async RabbitMQ message callback.
```
Comment (IrvingWash) [inline src/task.py]:
```suggestion
        This method is bound as an async RabbitMQ message callback.
```

**Pull request merged**